app/views/admin/attachments/_form.html.erb

```diff
@@ -41,7 +41,7 @@
         } %>
       </div>
 
-      <%= render "govuk_publishing_components/components/textarea", {
+      <%= render "components/govspeak-editor", {
```

Comment:
Sorry to be a pain but i'm getting this on this page when i click the preview button 

<img width="1704" alt="image" src="https://user-images.githubusercontent.com/42515961/198546139-165941b9-e748-4248-87b0-8af9d5b0e1f1.png">

Reply:
Is this the edit page? or some other page?

Reply:
Found the issue - pushing a fix